app/resources/websocket-user-settings/websocket-user-settings.js

```diff
@@ -0,0 +1,37 @@
+const javaScriptUtils = require("../../utils/javascript-utils/");
+
+class WebsocketUserSettings {
```

Comment:
This file contains business logic, it should not be a resource. A class in general is never a resource because it has state.

Reply:
Based on the comment, this restriction also applies to the [**TCC**](https://github.com/c-hive/team-contribution-calendar/blob/master/src/resources/TeamContributionCalendar/TeamContributionCalendar.js) class, doesn't it?

Reply:
Yes, good point. So resources are e.g. a simple JSON or SVG. Equivalent to assets in TCC's case. Everything else should be moved 1 level up.